### DISCUSSION TITLE
tibber-feed: does not connect

### DISCUSSION THREAD

**Discussion by mivola** [bisand/node-red-contrib-tibber-api#82] [General]
I'm using the tibber-feed but I cant get a proper connection. It does always just show "connecting":
![image](https://user-images.githubusercontent.com/743342/235782257-3bda8d30-c9b4-4b52-8a7d-35ef81b1bc7c.png)

In the log I can see: 
```
2 May 22:22:40 - [info] [tibber-feed:Home] Preparing to connect to Tibber... 
2 May 22:22:41 - [info] [tibber-feed:Home] Connecting to Tibber...
```

If I use the Demo-Token & Demo-Home it works though. And I can execute other requests with "tibber-data" successfully with my personal token, so it seems only the websocket subscription seems to fail. On the other hand I can see the liveMeasurements in the Api Explorer...

Any idea why that is and how to fix it?

Regards
Michael

**Comment by cmvbtx** [comment]
I have the exact same issue. Already changed the Token multiple times. My graphql query through a function is working like a charme, but the tibber node is always in state connecting. But used to work a couple of weeks ago.

**Comment by bisand** [comment]
Check if NodeRed's log on the server. If it works in the Tibber's API explorer, it should also work with NodeRed. And just in case, double check that you have enabled the live meassurement in your subscription.
It works for me...

**Reply by cmvbtx** [reply]
Thanks for the comment. I changed the API token and it started working again. I will observe the behavior. Thanks for your work!
